viper_go1_15.go

```diff
@@ -27,6 +27,13 @@ func (v *Viper) findConfigFile() (string, error) {
 
 func (v *Viper) searchInPath(in string) (filename string) {
 	v.logger.Debug("searching for config in path", "path", in)
+	
+	if v.configType != "" && stringInSlice(v.configType,SupportedExts) {
```

Comment:
Can you please provide some tests providing that it works and that it doesn't break previous behavior? (I believe it actually does thanks to that second condition)